kairon/shared/concurrency/actors/analytics_runner.py

```
@@ -0,0 +1,100 @@
+            process = subprocess.Popen(
+                [sys.executable, "-m", "kairon.shared.pyscript.analytics_worker"],
+                stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True
+            )
+
+            stdout, stderr = process.communicate(input=input_payload, timeout=600)
+
+            if process.returncode != 0:
+                raise AppException(f"Subprocess error: {stderr.strip()}")
```

Ruff (0.14.7)

Abstract `raise` to an inner function

(TRY301)